### ISSUE TITLE
Could `client_kwargs` be unconditional?

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-visualization#54]
```suggestion
        kwargs_zarr = {
            'storage_options': {'client_kwargs': request_init}
        }
```
For my own information, what would go wrong if we got rid of the `if`?

_Originally posted by @mccalluc in https://github.com/hubmapconsortium/portal-visualization/pull/53#discussion_r903920416_

**Comment by ilan-gold** [comment]
Hmmm not sure...kind of just thought it safe not to find out.  I imagine nothing though.

**Comment by mccalluc** [comment]
Resolved in #56
